models/public/anti-spoof-mn3/model.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Antispoofing model "anti-spoofing-mn3" based on the mobilenetv3 originally trained on PyTorch*
+  then converted to OpenVINO format.  For details about dataset see paper <https://arxiv.org/abs/2007.12342>,
+  repository <https://github.com/kirillProkofiev/light-weight-face-anti-spoofing>.
+task_type: classification
+files:
+  - name: anti-spoofing-mn3.onnx
```

Comment:
Is it possible to download the model in original PyTorch's format and convert to onnx with OMZ script? If so, it would be preferable.

Reply:
Maybe I do not understand what you mean, but the PyTorch model is just a dictionary of weights of the model's architecture. So to make conversion available I need to bring half of the project.

Reply:
You can use [pytorch_to_onnx.py](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/downloader/pytorch_to_onnx.py) script (examples [here](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/public/efficientnet-b0-pytorch/model.yml) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/public/single-human-pose-estimation-0001/model.yml)).

Reply:
@eizamaliev, the answer on your question in previous post "So to make conversion available I need to bring half of the project." model has modular structure and a lot of configuration parameters, it is hard to convert it using simple pytorch to onnx script